app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyDetailScreen.kt

```diff
@@ -85,6 +92,7 @@ fun SurveyDetailScreen(survey: Survey) {
             modifier = Modifier
                 .align(Alignment.BottomEnd)
                 .padding(bottom = 54.dp, end = 20.dp)
+                .semantics { contentDescription = startSurveyDescription }
```

Comment:
We should put this inside `StartSurveyBtn`, `SurveyDetailScreen` has no reason to access this 👀